staging/src/k8s.io/apiserver/pkg/server/options/authentication.go

```diff
@@ -394,3 +402,13 @@ func (s *DelegatingAuthenticationOptions) newTokenAccessReview() (authentication
 
 	return client.TokenReviews(), nil
 }
+
+func newIgnorableError(s string, args ...interface{}) ignorableError {
```

Comment:
simpler: `type ignorable error`. Then we don't need a constructor or any func. Just the type wrapper.

Reply:
If I'm creating just a type wrapper, then I wasn't able to differentiate between ignorable error and non-ignorable error.
See: https://play.golang.org/p/T0hSRsyPKj9

I'm fairly new to golang so this may be due to something I'm missing.

Reply:
Compare https://goplay.space/#C2IONvrPbiH